main.cpp

```diff
@@ -299,19 +363,30 @@ struct Receiver
     int function = 6;
     int inputs = 5;
     int outputs = 4;
+    Receiver();
     void playCds();
     void playRecords();
     void playRadio();
 };
 
+Receiver::Receiver()
+{
+    brand = "";
+    modelNum = 0;
+    function = 0;
+    inputs = 0;
+    outputs = 0;
```

Comment:
This overwrites the default values that your variables have. 
remove these value assignments.
- [ ] revise